### DISCUSSION TITLE
Wake up by command

### DISCUSSION THREAD

**Discussion by wedge76** [kube-green/kube-green#307] [Ideas]
On a testing cluster all our pods are running during working hours, Monday till Friday and sleep during the night.

In case I want to wake them all up for testing something (e.g. on Saturday), it would be great if this can be done by running a command.
Can you add this feature please?

**Comment by alexis974** [comment]
I second this idea ! Would be really useful especially for bench environment

**Comment by davidebianchi** [comment]
Hi! Thank you for the proposal. It's an interesting idea, we could expose some custom endpoint to do it.

**Reply by foxutech** [reply]
Thanks and it will be interesting to see.

**Reply by konrad0960** [reply]
@davidebianchi Hi there, I'm interested in contributing to open source projects that could help the environment. I'm rather junior level on golang/k8s and I just wanted to ask if its possible to get in touch with you to discuss on what I could possibly contribute to in this project. This particular topic with providing a command to start the pods seems really interesting to me. I'm sorry for contacting you that way but the only way to get in touch with you mentioned on GitHub is Twitter and I'm not a user of this platform.

**Reply by davidebianchi** [reply]
Hi @konradmolinski! I'm happy to hear that you are interested to contribute to the project! Yes, sure, you can reach me on LinkedIn!

**Comment by larssb** [comment]
Any news on this? Did the feature arrive?

**Comment by kinoute** [comment]
that would be a great feature if we have to wake up things for an emergency or for someone who wants to work in the off hours
